### PR TITLE
[Pages] no framework needed update

### PR DESCRIPTION
**Pull request by deadlypants1973**

### CODE REVIEW

File: content/pages/get-started/index.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ If using GitLab, you can select a project from your personal account or from a G
 
 {{<Aside type="note">}}
 
-If you do not have a project repository to choose from yet, refer to our extensive [How to section](/pages/how-to/) to learn how to create and deploy common frameworks and tools to Cloudflare Pages.
+If you do not have a project repository to choose from yet, refer to the [How to section](/pages/how-to/) to learn how to create and deploy common frameworks and tools to Cloudflare Pages.
```
Comment (WalshyDev) [inline content/pages/get-started/index.md]: We have a guide without a framework right? I think this little sentence too just indicates it needs a framework. Not sure what we'd rephrase it too though

Comment (deadlypants1973) [inline content/pages/get-started/index.md:47]:
```suggestion
honestly, i feel like this is unnecessary because none of the how to section addresses this at all. im going to delete this aside
```

**Review by WalshyDev — APPROVED** [review]

File: content/pages/framework-guides/deploy-anything.md [hunk 2]
```diff
@@ -25,11 +25,11 @@ Deploy your site to Pages by logging into the [Cloudflare dashboard](https://das
...
 
-| Configuration option     | Value              |
-| ------------------------ | ------------------ |
-| Production branch        | `main`             |
-| Build command (optional) | `yourbuildcommand` |
-| Build output directory   | `yourbuilddir`     |
+| Configuration option     | Value               |
+| ------------------------ | ------------------- |
+| Production branch        | `main`              |
+| Build command (optional) | `<YOURBUILDCOMMAND>`|
```
Comment (pedrosousa) [inline content/pages/framework-guides/deploy-anything.md]: Personally, I separate words in placeholders using underscores: `<YOUR_BUILD_COMMAND>`, but it's up to you.  :)

**Review by pedrosousa — APPROVED** [review]
